portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase

```diff
@@ -500,6 +500,60 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-78055. Liferay Admin user, for the virtual instance, is not able to login after the 'Required' box in LDAP configurations is turned on. ."
```

Comment:
Please update the descirption, according the ticket, what we test here is the happy path, not the negative aka original issue.

Reply:
Fixed.